learning/tour-of-beam/frontend/assets/translations/en.yaml

```diff
@@ -15,6 +15,18 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
+header:
+  signIn: 'Sign in'
```

Comment:
`header` and `footer` could be grouped into something like `ui`. Think of a layout change. Buttons can move between header, footer, and other places. We do not want to change translation keys because of that. Keys are better used for logical grouping.

`common` is also a good word, but I like to use it for things that are not tied to any single widget like "OK", "Cancel", "Save", "kB", "km/h", etc.